React/Views/RCTView.m

```diff
@@ -341,6 +361,136 @@ - (NSString *)accessibilityValue
   }
   return nil;
 }
+#else // [TODO(macOS ISS#2323203)
+- (id)accessibilityValue {
+  if (self.accessibilityRole == NSAccessibilityCheckBoxRole) {
+    for (NSString *state in self.accessibilityState) {
+      id val = self.accessibilityState[state];
+      if (!val) {
+        continue;
+      }
+      if ([state isEqualToString:@"checked"]) {
+        if ([val isKindOfClass:[NSNumber class]]) {
+          return [val boolValue] ? @YES : @NO;
+        } else if ([val isKindOfClass:[NSString class]] && [val isEqualToString:@"mixed"]) {
+          return @(2);
+        }
+      }
+    }
+  }
+
+  // handle accessibilityValue
+
+  if (self.accessibilityValueInternal) {
+    id now = self.accessibilityValueInternal[@"now"];
+    id text = self.accessibilityValueInternal[@"text"];
+    if (text && [text isKindOfClass:[NSString class]]) {
+      return text;
+    } else if ([now isKindOfClass:[NSNumber class]]) {
+      return now;
+    }
+  }
+
+  return nil;
+}
+
+- (BOOL)isAccessibilitySelectorAllowed:(SEL)selector {
+  if (selector == @selector(isAccessibilityEnabled)) {
+    if (self.accessibilityState) {
+      id disabled = self.accessibilityState[@"disabled"];
+      if ([disabled isKindOfClass:[NSNumber class]]) {
+        return YES;
```

Comment:
Just commenting once more here but early returns should be avoided https://github.com/microsoft/objc-guide/blob/7bc41b8608d55a696a5b1c44d05be0e7b2496954/MethodsAndImplementations/EarlyReturns.md